catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/NsiliConstants.java

```diff
@@ -38,6 +38,8 @@
                 return new ContentTypeImpl(type.getSpecName(), STANAG_VERSION);
             }));
 
+    public static final String NSIL_METACARD_TYPE = "NSILIMetacard";
```

Comment:
I thought the metacard type name was defined in blueprint as nsili.aggregate.  Is this needed?